newIDE/app/src/EventsSheet/ParameterFields/ObjectField.js

```diff
@@ -14,6 +14,41 @@ import { Trans } from '@lingui/macro';
 import { nameAndIconContainer } from '../EventsTree/ClassNames';
 import InAppTutorialContext from '../../InAppTutorial/InAppTutorialContext';
 
+const gd: libGDevelop = global.gd;
+
+export const getBehaviorConstraints = (
+  platform: gdPlatform,
+  functionMetadata: gdInstructionMetadata | gdExpressionMetadata,
+  parameterIndex: number
+) => {
+  const behaviorConstraints: Array<{
+    behaviorName: string,
+    behaviorType: string,
+  }> = [];
```

Comment:
Should you define a type that you can use a bit everywhere? I saw this in lots of places. And you can use the strict type `{| |}`

Reply:
I removed this part of the code.